SConstruct

```diff
@@ -826,7 +826,14 @@ if env["ARNOLD_ROOT"] :
 
 vTuneRoot = env.subst("$VTUNE_ROOT")
 
-gafferLib = {}
+if env["PLATFORM"] == "win32" : 
+	gafferLib = {
+		"envAppends" : {
+			"LIBS" : [ "Advapi32" ]
+		}
+	}
```

Comment:
Just noting that this'll get clobbered if anyone were to try to use VTUNE_ROOT on Windows. Perhaps the VTune support would be better split into a separate project anyway, and in the meantime it's unlikely to be used on Windows, so this is OK?

Reply:
How do you feel about https://github.com/GafferHQ/gaffer/pull/4524/commits/6a06feae8b6a9017921799435c357c36fe07c9b2, appending the libraries if they exist?

Reply:
I see what you mean. Have a look at https://github.com/GafferHQ/gaffer/pull/4524/commits/4924d8987f0ce915d575e3be308856d4a05e2988. I put both the Windows libraries and the vTune libraries after the main declaration, alongside the OpenGL and Qt handling. The need to initialize the lists makes it a bit awkward, but maybe it's an improvement?